main.go

```diff
@@ -21,29 +21,33 @@ var (
 	date        string
 	buildSource = "unknown"
 
-	configFlag    = flag.Bool("config", false, "Print the current default config")
-	debuggingFlag = flag.Bool("debug", false, "a boolean")
-	versionFlag   = flag.Bool("v", false, "Print the current version")
+	configFlag    = false
+	debuggingFlag = false
 )
 
 func main() {
-	flag.Parse()
-	if *versionFlag {
-		fmt.Printf("commit=%s, build date=%s, build source=%s, version=%s, os=%s, arch=%s\n", commit, date, buildSource, version, runtime.GOOS, runtime.GOARCH)
-		os.Exit(0)
-	}
 
-	if *configFlag {
+	flaggy.SetName("lazydocker")
+	flaggy.SetDescription("The lazier way to manage everything docker")
+	flaggy.DefaultParser.AdditionalHelpPrepend = "https://github.com/jesseduffield/lazydocker"
+
+	flaggy.Bool(&configFlag, "config", "config", "Print the current default config")
```

Comment:
I think that the shorthand for `config` should be `c`.

Reply:
Oops, thankyou!